pages/apply/[person]/[[...step]].tsx

```diff
@@ -0,0 +1,38 @@
+import ApplicationForms from "../../../components/application/application-forms"
+import Layout from "../../../components/layout/resident-layout"
+import { useRouter } from 'next/router'
+
+export default function ApplicationStep(): JSX.Element {
+  const router = useRouter()
+  let { person, step } = router.query
+  person = person as string
+
+  const baseHref = `/apply/${person}`
+  const steps: string[] = ["test-data", "immigration-status"]
```

Comment:
Define via props?

Reply:
This is the entry point (the page), so we pass it through to `<ApplicationForms />`